### DISCUSSION TITLE
我想问一下，买的你们的桌面版，有问题！！

### DISCUSSION THREAD

**Discussion by h3282** [AlistGo/alist#6655] [Q&A] [answered]
买的你们的桌面版，有问题，原来用了两个设备了，新买的迷你pc用最后一个设备，因系统有问题，重新装了一下系统，输入激活码，就出现已超过最大设备，这正常吗？

**Comment by anwen-anyi** [comment, marked as answer]
进行反激活设备
- **https://ad.nn.ci/zh/usage**

**Reply by h3282** [reply]
谢谢了！！弄好了。
